### PR TITLE
Add MockCifar10 to TF sanity tests

### PR DESCRIPTION
**Pull request by kshpv**

### CODE REVIEW

File: examples/tensorflow/classification/datasets/preprocessing_selector.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 PREPROCESSING_FN_MAP = {
     'imagenet2012': imagenet_preprocess_image,
     'cifar10': cifar10_preprocess_image,
+    'mock_cifar10': cifar10_preprocess_image,
```
Comment (alexsu52) [inline examples/tensorflow/classification/datasets/preprocessing_selector.py:24]: @kshpv, @andrey-churkin Could you provide the mock dataset for tests w/o changing the example code?

**Review by andrey-churkin — APPROVED** [review]

File: examples/tensorflow/classification/datasets/preprocessing_selector.py (hunk 1 above)
Comment (andrey-churkin) [reply examples/tensorflow/classification/datasets/preprocessing_selector.py]: Done

**Comment by andrey-churkin** [comment]
Seems like CI is still down. For this reason, we can't merge it.

**Comment by kshpv** [comment]
Jenkins please retry a build

**Comment by kshpv** [comment]
@alexsu52 please merge if satsified

**Pull request merged**